<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YaraEditor</class>
 <widget class="QMainWindow" name="YaraEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yara-Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../rsrc/yaraeditor.qrc">
    <normaloff>:/logo/images/logo.jpg</normaloff>:/logo/images/logo.jpg</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="MainWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widgetEditor" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1117</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidgetYara">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>208</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QWidget" name="widgetYara" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Yara Browser</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pathYara">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="yaraTree">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetMalware">
   <property name="minimumSize">
    <size>
     <width>310</width>
     <height>231</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Malware Browser</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QWidget" name="widgetMalware" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="pathMalware">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="malwareTree">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetInspector">
   <property name="windowTitle">
    <string>Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_properties">
        <attribute name="title">
         <string>Tab Properties</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QTreeWidget" name="treeMalwareProperties">
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_strings">
        <attribute name="title">
         <string>Strings</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7"/>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QTextEdit" name="outputEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNouveau">
   <property name="icon">
    <iconset>
     <normalon>:/icon/images/win/filenew.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../rsrc/yaraeditor.qrc"/>
 </resources>
 <connections/>
</ui>
